src/utils/blockchain/contracts.ts

```diff
@@ -107,6 +107,21 @@ export const contracts = {
     abi: LiquidityPoolV1Converter,
     blockNumber: 1218833,
   },
+  RIF_token: {
+    address: '0x19f64674d8a5b4e652319f5e239efd3bc969a1fe', // todo
+    abi: TestTokenABI,
```

Comment:
I know this doesn't have anything with this PR but why is the ABI called `TestTokenABI`? From what I've seen in the code, it's used for some "altcoins" that have just some common functionality. I would suggest renaming it to something more appropriate, such as `CommonTokenABI` or `GeneralTokenABI` (and rename the abi file itself as well).

Reply:
Indeed, it should be called `ERC20TokenABI`, but will make adjustments on another PR when all other PR's with this abi as dependency are merged